tests/components/tomato/test_device_tracker.py

```diff
@@ -68,9 +68,9 @@ def mock_session_send():
 def test_config_missing_optional_params(hass: HomeAssistant, mock_session_send) -> None:
     """Test the setup without optional parameters."""
     config = {
-        DOMAIN: tomato.PLATFORM_SCHEMA(
+        DEVICE_TRACKER_DOMAIN: tomato.PLATFORM_SCHEMA(
             {
-                CONF_PLATFORM: tomato.DOMAIN,
+                CONF_PLATFORM: DEVICE_TRACKER_DOMAIN,
```

Comment:
`tomato.DOMAIN` would refer to `homeassistant.components.tomato.device_tracker.DOMAIN`, which in turns referred to `homeassistant.components.device_tracker.DOMAIN`
This has now been simplified to `DEVICE_TRACKER_DOMAIN`

Maybe the intention was to use `"tomato"` and not `"device_tracker"` but I decided to match the existing string value.

Reply:
This seems very strange. None of these tests would have worked?

Reply:
For now - I'm using the same backend string value as it was before - so no change in the tests.
I only touched the tests because the constant was aliased and so didn't exist anymore.